Assets/Scripts/Interactables/Interactable.cs

```diff
@@ -15,7 +15,7 @@ public class Interactable : MonoBehaviour, IInteractable
 
 
     // Start is called before the first frame update
-    protected void Start()
+    public virtual void Start()
     {
         interactableAnim = GetComponentInChildren<Animator>();
         interactTransform = GetComponentInChildren<RectTransform>();
```

Comment:
I removed these in my recent PR to be components that you could just drag into the inspector. You can remove these to resolve the conflict.

Reply:
Ooh! Wait why did you do that? Isn't it easier if the user doesn't have to drag more stuff when they set it up?

Reply:
I can remove though!

Reply:
I'd wait for Chadley's updates in main and then updating and testing your stuff again, just to make sure!

Reply:
I merged my changes to master so it should be good to go! @justineechen Let me know if there are any issues once updated

Reply:
great! So i just copied your file from master and then made my tweaks to it so i hope there won't be any big issuess!